### ISSUE TITLE
Images loading all at init

### ISSUE THREAD

**Issue by vivoconunxino** [eisbehr-/jquery.lazy#70]
Does this exact script run in your localhost? It's not working properly on mine

```

<html>
    <head>
        <script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/3.0.0/jquery.js"></script>
        <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery.lazy/1.7.4/jquery.lazy.min.js"></script>

        <style>
            img.lazy {
                width: 700px;
                height: 467px;
                display: block;
            }
        </style>
    </head>

    <body>

        <script type="text/javascript" charset="utf-8">
            $(function() {
                var loadedElements = 0;
                $('img.lazy').lazy({
                        scrollDirection: 'vertical',
                        effect: 'fadeIn',
                        visibleOnly: true,
                        onError: function(element) {
                            console.log('error loading ' + element.data('src'));
                        },
                        beforeLoad: function(element) {
                            console.log('image "' + element.data('src') + '" is about to be loaded');
                        },
                        afterLoad: function(element) {
                            loadedElements++;
                            console.log('image "' + element.data('src') + '" was loaded successfully');
                        },
                        onFinishedAll: function() {
                            console.log('finished loading ' + loadedElements + ' elements');
                            console.log('lazy instance is about to be destroyed')
                        }
                });
            });   
        </script>


        <br /><br /><br /><br /><br /><br /><br /><br /><br />
        <br /><br /><br /><br /><br /><br /><br /><br /><br />
        <br /><br /><br /><br /><br /><br /><br /><br /><br />
        <br /><br /><br /><br /><br /><br /><br /><br /><br />
        <br /><br /><br /><br /><br /><br /><br /><br /><br />

        <h1>Images</h1>
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/0.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/1.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/2.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/3.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/4.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/5.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/6.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/7.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/8.jpg" src="" />
        <img class="lazy" height="490" data-src="http://jquery.eisbehr.de/lazy/images/9.jpg" src="" />
    </body>
</html>
```

**Comment by vivoconunxino** [comment]
There must be something wrong here Daniel. Almost the same code but with tuupola's lazyload works perfectly.

**Comment by eisbehr-** [comment]
Yes, there is a problem with your document, and yes, it's not working for me too. But this is only because your html document is invalid. You missed to add the `doctype` to the content. When you adding this everything workes fine ...

```HTML
<!DOCTYPE html>
<html>
  <head>
    ...
```

**Comment by vivoconunxino** [comment]
I did what you say and it works, now I tried to use your plugin with a page in which I work with an external designer and it doesn't... it still loads all the elements at once, this time there is a doctype element but there probably are also some html/css errors. I imagine that it all was done in a perfect standard way it would work perfectly. The thing is that I can't review all his design (and it's out of my scope and knowledge) and I tried the another plugin I mentioned firstly and it works as expected.

I like yours since it has a lot more of capabilities but I guess it's a bit more restrictive with the html (css?) syntax used.

Well, just to let you know. It's obvious you've put a lot of effort into this and I would ensure you also put into everything you do. Happy new year.
